adafruit_httpserver/request.py

```diff
@@ -33,8 +33,35 @@ def _add_field_value(self, field_name: str, value: Union[str, bytes]) -> None:
         else:
             self._storage[field_name].append(value)
 
-    def get(self, field_name: str, default: Any = None) -> Union[str, bytes, None]:
+    @staticmethod
+    def _html_output_encode(value):
+        """Encodes unsafe HTML characters."""
+        return (
+            str(value)
+            .replace("&", "&amp;")
+            .replace("<", "&lt;")
+            .replace(">", "&gt;")
+            .replace('"', "&quot;")
+            .replace("'", "&#x27;")
```

Comment:
```suggestion
            .replace("'", "&apos;")
```

Reply:
Thank you. I assumed that one didn't exist since it was the "odd one out" in the resource I copied from. Changed with latest commit.